root_VS2013/programs/VB/Samples/WebApp_sample/ProjectX_sample/ProjectX_sample.vbproj

```diff
@@ -15,11 +15,6 @@
     <AssemblyName>ProjectX_sample</AssemblyName>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <MyType>Custom</MyType>
-    <UseIISExpress>false</UseIISExpress>
-    <IISExpressSSLPort />
-    <IISExpressAnonymousAuthentication />
-    <IISExpressWindowsAuthentication />
-    <IISExpressUseClassicPipelineMode />
   </PropertyGroup>
```

Comment:
@yusukemaegawa 

I have remove only the added items in this time.

- [Fixed : "To merge the changes ( - Until September 30)." #185](https://github.com/OpenTouryoProject/OpenTouryoTemplates/pull/57)
  - [Fixed Json service changes #181](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/181)

The check result in current state, it has been configured such as following.

- All *.csproj is set as "\<UseIISExpress\>true\</UseIISExpress\>"
  - https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2012/programs/C%23/Samples/WebApp_sample/ProjectX_sample/ProjectX_sample.csproj#L17
  - https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2013/programs/C%23/Samples/WebApp_sample/ProjectX_sample/ProjectX_sample.csproj#L17
  - https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2015/programs/C%23/Samples/WebApp_sample/ProjectX_sample/ProjectX_sample.csproj#L17
- All *.vbproj is set as "\<UseIISExpress\>false\</UseIISExpress\>"
  - https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2012/programs/VB/Samples/WebApp_sample/ProjectX_sample/ProjectX_sample.vbproj#L18
  - https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2013/programs/VB/Samples/WebApp_sample/ProjectX_sample/ProjectX_sample.vbproj#L18
  - https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2015/programs/VB/Samples/WebApp_sample/ProjectX_sample/ProjectX_sample.vbproj#L18

If the answer for your question, we need to consider what should be this setting.